packages/itmat-interface/test/serverTests/log.test.ts

```diff
@@ -43,6 +44,9 @@ afterAll(async () => {
 });
 
 beforeAll(async () => { // eslint-disable-line no-undef
+    /* Choose API version used in this test file */
+    apiPath = '/api/'.concat(config.apiVersions[config.apiVersions.length - 1]);
```

Comment:
When testing we still need to verify the previous versions are still passing there tests, otherwise this might lead us into error when older software uses them and they break everything.

Reply:
As there must be differences between the same queries from different api version, what I use to suppose is that a single test file only test one api versioning to avoid conflicts. So there might be a logV1.test.ts and logV0.test.ts at the same time.